simcon_project/conversation_templates/views/conversation.py

```diff
@@ -0,0 +1,139 @@
+from django.shortcuts import render, redirect
+from django.http import HttpResponseNotFound
+from django.forms import modelformset_factory
+from conversation_templates.models import ConversationTemplate, TemplateNode, TemplateNodeResponse, TemplateResponse
+from conversation_templates.forms import TemplateNodeChoiceForm
+from users.models import Student, Assignment
+from datetime import datetime
+from django.contrib.auth.decorators import login_required
+import django_tables2 as tables
+
+# Globals
+ct_templates_dir = 'conversation'
+
+
+class NodeDescriptionTable(tables.Table):
+    """
+    Creates table that displays the description for each TemplateNode object that a Student visited.
+    """
+    node_description = tables.Column()
+
+
+# Views
+@login_required(login_url='/accounts/login/')
```

Comment:
You don't have to provide it manually since it can be set in the settings.py (which is what I did in my PR if you wanna check it out). It's good to keep that stuff in one place so that it reflects everywhere if it changes.

Reply:
Cool I'll remove the path

Reply:
@ericharper1 oops i pushed up this change on the researcher view, could you fix it or i can adjust on my next PR